src/libpriv/rpmostree-postprocess.c

```diff
@@ -1893,8 +1908,15 @@ rpmostree_commit (int            rootfs_fd,
         return FALSE;
     }
 
-  if (!ostree_repo_prepare_transaction (repo, NULL, cancellable, error))
-    return FALSE;
+  /* See comment above */
+  const gboolean use_txn = getenv ("RPMOSTREE_COMMIT_NO_TXN") == NULL &&
```

Comment:
could we possibly add this to the docs somewhere?

Reply:
I think this is a temporary hack; the right solution is bare-user repos and pull-local, and given our use of detecting NFS, not necessary for Fedora at least.